src/screens/Login/index.tsx

```diff
@@ -1,7 +1,56 @@
-import React from 'react';
+import React, { useState } from 'react';
+import i18next from 'i18next';
+import { useMutation } from 'react-query';
+import { useHistory } from 'react-router-dom';
+
+import Loading from 'components/Spinner/components/loading';
+import LoginForm from 'components/Forms/LoginForm';
+import { UserCredentials } from 'utils/types';
+import { login } from 'services/UsersService';
+
+import styles from './styles.module.scss';
 
 function Login() {
-  return <div>Login component works!</div>;
+  const history = useHistory();
+  const [errorMsg, setErrorMsg] = useState('');
+  const { isLoading, isError, mutate } = useMutation((data: UserCredentials) => login(data), {
+    onSuccess: res => {
+      // eslint-disable-next-line no-console
+      console.log('access-token: ', res);
+      history.push('/home');
+    },
+    onError: () => {
+      setErrorMsg(i18next.t('FormValidations:credentials'));
+    }
+  });
+  const onValid = (userCredentials: UserCredentials) => {
+    mutate(userCredentials);
+  };
+  const onSignUpButtonClick = () => {
+    history.push('/sign_up');
+  };
+
+  return (
+    <div className={styles.container}>
```

Comment:
here can you use class like at column, center and you don't need to add them in the styles

`.container {
```suggestion
    <div className={`column ${styles.container}`}>
```
  height: 100%;
  position: absolute;
  width: 100%;
}`

Reply:
It was great advice.
It allow me to make several adjustments related to styles, especially I created classes in the `layout.scss `file and refactor the way how the screens are using the styles.
Thanks!